hr_expense_sequence/views/hr_expense_expense_view.xml

```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Same question, why this removal was needed?

Reply:
@alexey-pelykh Sorry, I misunderstand. Edied.